### DISCUSSION TITLE
Suggest similar existing issues when typing the title for a new one

### DISCUSSION THREAD

**Discussion by douglaslassance** [community/community#8277] [Projects and Issues]
Suggesting similar existing issues when typing the title of a new one would guard against duplication and help guide users to the right place.

**Comment by Strajk** [comment]
Example how YouTrack does this

![image](https://user-images.githubusercontent.com/697301/162990833-3092d28d-3d49-413e-bb58-117ce52b0558.png)

**Comment by vict0rsch** [comment]
I think this would be particularly useful for orgs like mine who want to use Github Discussions in a Q&A / StackOverflow form. Reducing redundancy with the same kind of title-based deduplication **before** the discussion/issue is actually opened would be amazing

**Comment by pimterry** [comment]
As an open source maintainer, I'd find this _extremely_ useful. Lots of people file exact (word for word!) duplicates of existing issues, and it wastes substantial amounts of my time and theirs. Everybody would win if there was an obvious prompt towards similar existing issues.

**Comment by corneliusroemer** [comment]
I wrote a detailed feature proposal here (ironically I only noticed this issue here exists after I wrote mine, case in point!): https://github.com/orgs/community/discussions/137345

Another related discussion is: https://github.com/orgs/community/discussions/18185

Gitlab does it like that:
![2024-08-31 16 40 03](https://github.com/user-attachments/assets/f7834f6b-ffdf-4ac0-857c-b7c449d997a4)
